Readability.js

```diff
@@ -1655,8 +1655,6 @@ Readability.prototype = {
           toRemove = true;
         } else if (!isList && weight < 25 && linkDensity > 0.2) {
           toRemove = true;
-        } else if (weight >= 25 && linkDensity > 0.5) {
-          toRemove = true;
```

Comment:
> This change affects more than just list items, as evidenced by the buzzfeed testcase. I think we need to tread slightly more carefully here.

Sure, but it fixes a previous actually broken test case (missing image), add support for a new one, without breaking anything else. Also, this rules doesn't make much sense to me… removing elements with some weight but having some large link density doesn't really make sense to me. Don't forget we've already extracted our top candidate node at this point, so things like menus and navigation entries has been stripped out already…

Do you have a specific real-world use case in mind?